tldw_Server_API/app/api/v1/endpoints/audiobooks.py

```
@@ -0,0 +1,453 @@
+        status=_normalize_job_status(job.get("status")),
+    )
+
+
+@router.get(
+    "/jobs/{job_id}",
+    response_model=AudiobookJobStatusResponse,
+    summary="Get audiobook job status",
+)
+async def get_audiobook_job_status(
+    job_id: int = Path(..., ge=1, description="Audiobook job id"),
+    _current_user: User = Depends(get_request_user),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)